syne_tune/backend/local_backend.py

```diff
@@ -195,7 +205,14 @@ def _all_trial_results(self, trial_ids: List[int]) -> List[TrialResult]:
             # (which allows to write a time-stamp when the process finishes) but it is probably OK if all_results
             # is called every few seconds.
             if os.path.exists(trial_path / "end"):
-                training_end_time = self._read_time_stamp(trial_id=trial_id, name="end")
+                try:
+                    training_end_time = self._read_time_stamp(
+                        trial_id=trial_id, name="end")
+                except ValueError as exc:
```

Comment:
Same question as bellow, why is this one needed?

Reply:
I had this exception being thrown. I can take this out, it was probably again due to disk filling up.

But could it not happen that the file is written, but empty or corrupted? The code here would then continue and treat it as if it is not there.

Reply:
I would rather remove this catch because it does not seem we can recover the tuning if files cannot be written.